dbt_platform_helper/commands/conduit.py

```diff
@@ -222,15 +223,29 @@ def create_addon_client_task(
     elif addon_type == "redis" or addon_type == "opensearch":
         secret_name += "_ENDPOINT"
 
-    subprocess.call(
-        f"copilot task run --app {app.name} --env {env} "
-        f"--task-group-name {task_name} "
-        f"--image {CONDUIT_DOCKER_IMAGE_LOCATION}:{addon_type} "
-        f"--secrets CONNECTION_SECRET={get_connection_secret_arn(app, env, secret_name)} "
-        "--platform-os linux "
-        "--platform-arch arm64",
-        shell=True,
-    )
+    try:
+        session.client("iam").get_role(RoleName=f"{app.name}-{addon_type}-{app.name}-{env}-ecsTask")
```

Comment:
if this is the line that's meant to raise the exception, then we could clean this up a bit to not repeat the `subprocess.call(...)` block?

we could do something like

```
try:
    session.client("iam").get_role(RoleName=f"{app.name}-{addon_type}-{app.name}-{env}-ecsTask")
    execution_role = "--execution-role {app.name}-{addon_type}-{app.name}-{env}-ecsTask "
except:
    execution_role = ""

suprocess.call(...
   f"{execution_role}"
)
```

Reply:
Reminder: you'll need to change this if you action Will's comment in the other PR

Reply:
Thanks for the reminder. Both sides updated!